src/coreclr/clrdefinitions.cmake

```diff
@@ -179,8 +179,10 @@ if(FEATURE_MERGE_JIT_AND_ENGINE)
 endif(FEATURE_MERGE_JIT_AND_ENGINE)
 add_compile_definitions($<$<NOT:$<BOOL:$<TARGET_PROPERTY:CROSSGEN_COMPONENT>>>:FEATURE_MULTICOREJIT>)
 if(CLR_CMAKE_TARGET_UNIX)
-  add_definitions(-DFEATURE_PAL)
+  add_definitions(-D_TARGET_UNIX_)
```

Comment:
Please see my comment about identifiers begining with an underscore. (They're UB in C.)

Reply:
@lpereira I revised the patch try to fix many of the identifiers with an underscore.